app/api/images/images.controller.js

```diff
@@ -482,7 +513,8 @@ exports.submitImage = utils.route(async (req, res) => {
     date_inserted: models.sequelize.literal("current_timestamp"),
     exact_date: exact_date_req,
     iiif_data: {
-      image_service3_url: req.body.iiif_link
+      image_service3_url: req.body.iiif_data.image_service3_url,
+      regionByPx: req.body.iiif_data.regionByPx ? req.body.iiif_data.regionByPx.toString() : null
```

Comment:
actually would probably make more sense to store the region as an array (to be later returned also by the API) and to use `array_to_string` to create the image url in the SQL query